.rubocop.yml

```diff
@@ -282,5 +288,12 @@ Style/HashTransformKeys:
 Style/HashTransformValues:
   Enabled: true
 
-Rspec/MultipleMemoizedHelpers:
+
+RSpec/MultipleMemoizedHelpers:
+  Max: 15
+
+RSpec/MultipleExpectations:
   Max: 15
+
+RSpec/ExampleLength:
+  Max: 25
```

Comment:
In my attempt to bring you a github_integration without rubocop warnings, I stumbled over some RSpec-related rules which didn't have good defaults yet (in my opinion, of course).

I'd suggest these as relaxed good defaults as a starting point. We can probably adjust these to our needs over time.

Reply:
:+1: